core/src/solvers/cartesian_path.cpp

```diff
@@ -54,7 +54,7 @@ CartesianPath::CartesianPath() {
 	auto& p = properties();
 	p.declare<geometry_msgs::PoseStamped>("ik_frame", "frame to move linearly (use for joint-space target)");
 	p.declare<double>("step_size", 0.01, "step size between consecutive waypoints");
-	p.declare<double>("jump_threshold", 1.5, "acceptable fraction of mean joint motion per step");
+	p.declare<moveit::core::CartesianPrecision>("precision", { 1e-3, 1e-2, 1e-4 }, "precision of linear path");
```

Comment:
Why explicitly provide values here and not rely [on the defaults](https://github.com/moveit/moveit/blob/026c512b12e1723630deef9bf810066d7e95641c/moveit_core/robot_state/include/moveit/robot_state/cartesian_interpolator.h#L50-L55)? And why is the third value different from MoveIt's default of `1e-5`?